src/components/widgets/toolhead/ToolheadCard.vue

```diff
@@ -154,8 +154,10 @@ type Tool = {
   name: string,
   label?: string,
   disabled?: boolean,
+  enabledOnPrint?: boolean,
   wait?: string,
   icon?: string,
+  callback?: () => void,
 }
```

Comment:
Are these changes required? I can't seem to find this in use anywhere...

Reply:
Oops, that's a leftover from when the Change Spool button was in the Tools submenu, I'll remove those later